_info/Tribute

```diff
@@ -1,7 +1,7 @@
 Tribute
 
-Every day phase, each team's king has to choose to tribute one of this team members. This is mandatory. They have to also choose a certain amount of money to assign to the person. 
-The lowest paying team will have their tribute killed. Use `!tribute` to set your tribute candidate if you're the king. 
+During each day phase, the team's kings have to choose to tribute one of their team members, with a certain amount of money assigned to the tribute. This is mandatory. The lowest paying team will have their tribute killed. 
```

Comment:
"each team" is better than "the team" because there are multiple teams
also the second comma shouldn't be there

Reply:
Will fix